plugins/techdocs/src/reader/components/Reader.tsx

```diff
@@ -64,6 +64,10 @@ const useStyles = makeStyles<BackstageTheme>(theme => ({
     marginLeft: '20rem',
     maxWidth: 'calc(100% - 20rem * 2 - 3rem)',
     marginTop: theme.spacing(1),
+    '@media screen and (max-width: 76.1875em)': {
```

Comment:
As its using the same width of the breakpoint as the injectCSS transformer https://github.com/backstage/backstage/blob/master/plugins/techdocs/src/reader/components/Reader.tsx#L171 when the secondary sidebar gets hidden. I think its fine to justify the calculation so that it only gives space to one sidebar (to the left).